src/encoded/schemas/individual.json

```diff
@@ -54,6 +54,37 @@
                 "mixed"
             ]
         },
+        "individual_relation": {
+            "title": "Individual Relations",
+            "description": "Related individuals",
+            "type": "array",
```

Comment:
You can add here:
"uniqueItems": true,

It doesn't change the need for the check in the validator that looks for different relationship types for the same individual but it will prevent mistakes in submitting multiple of the same individual_relations objects.

Reply:
OK!
And is there a way in the schema to limit the number of individual_relation items to max 2?

Reply:
You can try:

"maxItems": 2,

But that might have either been introduced in a later version of json schema than we are using or require some code someplace in snovault to do that type of validation.  Worth a try though.